### PR TITLE
Add schema for quanta firmware update tasks and remove null options

### PR DESCRIPTION
**Pull request by pengz1**
This PR is to add SKU schema verification for quanta firmware update tasks, and remove null options in quanta SKU workflows and dell RACADM related tasks.

Related PR:
https://github.com/RackHD/on-tasks/pull/329

### CODE REVIEW

**Comment by pengz1** [comment]
@RackHD/corecommitters @cgx027 @WangWinson

File: quanta-d51-2u/README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## BIOS firmware upgarde
 - The BIOS firmware files should be acquired from the vendor.  The BIOS image and BIOS upgrade executable are then extracted into the /static/bios location of the skupack and the config.json should be updated with the md5sum of the firmware image.
+- A valid BMC or BIOS firmware image should end with .bin, .BIN, .zip, .img or .ima, and only image path with bios/, bmc/ folder or without folder are supported. Other format of "file" value will be recognized as fault by schema validator.
```
Comment (yyscamper) [inline quanta-d51-2u/README.md]: I have some concern this is not be a good idea to restrict the image path.
Add @zyoung51 for confirm, whether this kind of restriction matches with your design?

**Comment by pengz1** [comment]
@cgx027

File: quanta-d51-2u/README.md (hunk 1 above)
Comment (cgx027) [reply quanta-d51-2u/README.md]: Adding examples here may help people to understand the image path requirement.

**Comment by cgx027** [comment]
With minor change suggestions but still :+1:

**Comment by yyscamper** [comment]
:+1:

**Comment by WangWinson** [comment]
:+1:

**Pull request merged**